src/peft/peft_model.py

```diff
@@ -192,6 +193,12 @@ def save_pretrained(
             is_main_process (`bool`, *optional*):
                 Whether the process calling this is the main process or not. Will default to `True`. Will not save the
                 checkpoint if not on the main process, which is important for multi device setups (e.g. DDP).
+            convert_pissa_to_lora (`str`):
+                The initial path for the PISSA adapter.
```

Comment:
I think we have to be more precise here when it comes to what is expected. When I reviewed the PR, I was still confused (maybe that's just me :D). Let's mention here that this should be the adapter that is obtained _after_ initializing the model with PiSSA and _before_ performing any training.